packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/diagrams/EmptyDocumentStereotypeHandler.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.services.diagrams;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.emf.ResourceMetadataAdapter;
+import org.eclipse.sirius.components.emf.migration.MigrationService;
+import org.eclipse.sirius.components.emf.migration.api.IMigrationParticipant;
+import org.eclipse.sirius.components.emf.services.JSONResourceFactory;
+import org.eclipse.sirius.components.emf.services.api.IEMFEditingContext;
+import org.eclipse.sirius.web.application.document.dto.DocumentDTO;
+import org.eclipse.sirius.web.application.document.services.api.IStereotypeHandler;
+import org.eclipse.sirius.web.application.views.explorer.services.ExplorerDescriptionProvider;
+import org.springframework.stereotype.Service;
+
+/**
+ * IStereotypeHandler test allowing to create an empty document.
+ *
+ * @author arichard
+ */
+@Service
+public class EmptyDocumentStereotypeHandler implements IStereotypeHandler {
```

Comment:
Sorry I forgot to remove this file, it is useless with the latest version of the PR.